main.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
@krishchow To build the container image using release tools, this main.go needs to be moved to `cmd/csi-adapater/main.go`

Reply:
Does anything else need to change (aside from updating references) in the main.go file for the release tools?